Update tip for locked state tool versions (#1388)
ActiveState/cli: changelog.md

- We've replaced the auto updating system with a manual updating system, updates
  will from now on be made available through the system tray application and you
  will have the choice to install it at your own convenience.

> - Added a flag `--set-version` to the `state update` command to update to a
>   specific State Tool version.  ([PR #1385](https://github.com/ActiveState/cli/pull/1385))

Reviewer: ```suggestion
- You can now switch to specific State Tool versions by running `state update --set-version <version>` ([PR #1385](https://github.com/ActiveState/cli/pull/1385))
```
